stuff/process_badges.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/python3
+# pylint: disable=invalid-name
+# This program is intended to be invoked from the console, not to be used as a
+# module.
+
+'''
+Script for the validation and processing of badges.
+'''
+
+
+import argparse
+import json
+import logging
+import os
+
+
+OMEGAUP_BADGES_ROOT = (os.path.abspath(os.path.join(__file__, '..', '..'))
+                       + '/frontend/badges/')
+
+
+def process_badge(alias):
+    '''Validates and processes badge information'''
+    logging.info('BADGE %s:', alias)
+    try:
+        path = OMEGAUP_BADGES_ROOT + alias + '/icon.svg'
+        filesize = os.stat(path).st_size
+        if filesize / 1024 > 15.0:
```

Comment:
qué tal

```suggestion
        if filesize  > _MAX_BADGE_SIZE:
```

?

y luego 

```python
_MAX_BADGE_SIZE = 15 * 1024
```

hasta arriba.